dev-libs/mpdecimal/mpdecimal-4.0.0.ebuild

```diff
@@ -43,7 +43,7 @@ src_prepare() {
 src_configure() {
 	local myconf=(
 		# just COPYRIGHT.txt
-		--docdir=/removeme
+		--docdir="${EPREFIX}"/removeme
```

Comment:
Thanks. However, could you try changing the `rm` below to use `${D}`  rather than `${ED}` instead? That should be effectively less roundabout.

Reply:
Done, thanks!